tests/testTriggers.go

```diff
@@ -3735,6 +3763,178 @@ func TriggerPoolAddDiskAndReboot(contexts *[]*scheduler.Context, recordChan *cha
 
 }
 
+func TriggerAutopilotPoolRebalance(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer ginkgo.GinkgoRecover()
+	event := &EventRecord{
+		Event: Event{
+			ID:   GenerateUUID(),
+			Type: AutopilotRebalance,
+		},
+		Start:   time.Now().Format(time.RFC1123),
+		Outcome: []error{},
+	}
+
+	defer func() {
+		event.End = time.Now().Format(time.RFC1123)
+		*recordChan <- event
+	}()
+	poolLabel := map[string]string{"autopilot": "rebalance"}
+	apRule := aututils.PoolRuleRebalanceAbsolute(120, 70, false)
```

Comment:
Can we avoid hardcoded value here?

Reply:
these are standard values mentioned in docs. I dont think this needs to be parameterized

Reply:
Can we define these as constant and use them ?